Show return code from libusb_submit_transfer in error message

--- libuvc-0.0.5/src/stream.c.old	2016-11-02 12:03:37.198166775 +0000
+++ libuvc-0.0.5/src/stream.c	2016-11-02 12:04:41.126709436 +0000
@@ -985,7 +985,7 @@
       transfer_id++) {
     ret = libusb_submit_transfer(strmh->transfers[transfer_id]);
     if (ret != UVC_SUCCESS) {
-      UVC_DEBUG("libusb_submit_transfer failed");
+      UVC_DEBUG("libusb_submit_transfer failed: %d",ret);
       break;
     }
   }
